integtest/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -0,0 +1,139 @@
+                    .pollPeriod(Duration.ofMillis(50))
+                    .workflowNumRetry(3)
+                    .build();
+        }
+    }
+
+    @Builder
+    @Getter
+    class MultipleServersFixture implements Fixture<ImmutableList<ServerParams>> {
+        @Default
+        private final int numNodes = 3;
+
+        @Override
+        public ImmutableList<ServerParams> data() {
+            List<ServerParams> serversParams = new ArrayList<>();
+
+            for (int i = 0; i < numNodes; i++) {
+                final int port = 9000 + i;
+                Mode mode = i == 0 ? Mode.SINGLE : Mode.CLUSTER;
+
+                ServerParams serverParam = SingleServerFixture
+                        .builder()
+                        .port(port)
+                        .mode(mode)
+                        .build()
+                        .data();
+
+                serversParams.add(serverParam);
+            }
+            return ImmutableList.copyOf(serversParams);
+        }
+    }
+
+    @Builder
+    @Getter
+    class CorfuServiceFixture implements Fixture<ServiceParams<ServerParams>> {
+        @Default
+        private final MultipleServersFixture servers = MultipleServersFixture.builder().build();
+        @Default
+        private final String serviceName = "corfuServer";
+
+        @Override
+        public ServiceParams<ServerParams> data() {
+            return ServiceParams.<ServerParams>builder()
+                    .name(serviceName)
+                    .nodes(ImmutableList.copyOf(servers.data()))
+                    .build();
+        }
+    }
+
+    @Builder
+    @Getter
+    class ClusterFixture implements Fixture<ClusterParams> {
+        @Default
+        private final CorfuServiceFixture service = CorfuServiceFixture.builder().build();
+
+        @Override
+        public ClusterParams data() {
+            ServiceParams<ServerParams> serviceParams = service.data();
+            return ClusterParams.builder()
+                    .services(ImmutableMap.of(serviceParams.getName(), serviceParams))
+                    .build();
+        }
+    }
+
+    @Builder
+    @Getter
+    class SingleClusterFixture implements Fixture<ClusterParams> {
+        @Default
+        private final CorfuSingleServiceFixture service = CorfuSingleServiceFixture.builder().build();
+
+        @Override
+        public ClusterParams data() {
+            ServiceParams<ServerParams> serviceParams = service.data();
+            return ClusterParams.builder()
+                    .services(ImmutableMap.of(serviceParams.getName(), serviceParams))
+                    .build();
+        }
+    }
+
+    @Builder
+    @Getter
+    class CorfuSingleServiceFixture implements Fixture<ServiceParams<ServerParams>> {
+
+        @Default
+        private final SingleServerFixture servers = SingleServerFixture.builder().build();
+        @Default
+        private final String serviceName = "corfuServer";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)